### ISSUE TITLE
detect WebXR API (successor to WebVR v1.1)

### ISSUE THREAD

**Issue by cvan** [TimvanScherpenzeel/detect-features#2]
First off, this is an awesome library! Thanks for making and sharing this.

It's similar to a small library we built for our [Mozilla Games](https://games.mozilla.org/) initiative: [`browser_feature_test`](https://github.com/Mozilla-Games/browser_feature_test) (**[demo here](https://mozilla-games.github.io/browser_feature_test/example.html)**)

<hr>

Anyway, [WebXR](https://immersive-web.github.io/webxr/spec/latest/) (VR + AR) is the new experimental API that will be soon superceeding [WebVR](https://immersive-web.github.io/webvr/spec/1.1/).

Here's a helpful blog post by @toji (Chrome engineer on WebXR + WebGL + Gamepad) on the experimental WebXR support behind a feature flag in Chrome Canary: https://blog.tojicode.com/2018/02/early-access-to-webxr-device-api-in.html

And here are the two specs:

- https://immersive-web.github.io/webvr/spec/1.1/ — [`navigator.getVRDisplays`](https://immersive-web.github.io/webvr/spec/1.1/#interface-navigator), [`VRDisplay`](https://immersive-web.github.io/webvr/spec/1.1/#interface-vrdisplay), [etc.](https://immersive-web.github.io/webvr/spec/1.1/#interfaces)
- https://immersive-web.github.io/webxr/spec/latest/ — [`navigator.xr`](https://immersive-web.github.io/webxr/spec/latest/#xr-interface), [`XRDevice`](https://immersive-web.github.io/webxr/spec/latest/#xrdevice-interface), [etc.](https://immersive-web.github.io/webxr/spec/latest/#session)

Long story short: for WebXR, we just need to check for truthiness of `navigator.xr` as opposed to `navigator.getVRDisplays`.

Many thanks for this library and for your help going forward with this library!

**Comment by TimvanScherpenzeel** [comment]
Hi,

Thank you for your pull request! Sounds like a great idea.

**Comment by TimvanScherpenzeel** [comment]
Merged 👍
